packages/app/src/renderer/wrappers/root-wrapper.tsx

```diff
@@ -148,6 +148,19 @@ const RootWrapper: FunctionComponent<Props> = ({
     connect()
   }, [])
 
+  useEffect(() => {
+    let interval: NodeJS.Timeout
+
+    if (connectedAndUnlocked) {
+      getCrashDump()
+      interval = setInterval(() => updateAllIndexes(), 30000)
```

Comment:
```suggestion
```

Reply:
Maybe a feature flag would be better in this case

Reply:
@igorbezsmertnyi what do you mean? You would like to add a feature flag for crashDump logic? 🤔

Reply:
@dkarski I mean this line `interval = setInterval(() => updateAllIndexes(), 30000)`

Reply:
this line is deprecated 😛